tests/ecosystem/deployment/test_deployment.py

```diff
@@ -15,9 +16,139 @@ def test_deployment():
     deploy = config.RUN['cli_params'].get('deploy')
     teardown = config.RUN['cli_params'].get('teardown')
     if not teardown or deploy:
-        assert is_cluster_running(config.ENV_DATA['cluster_path'])
+        ocs_install_verification()
 
     if teardown:
         log.info(
             "Cluster will be destroyed during teardown part of this test."
         )
+
+
+def ocs_install_verification():
+    """
+    Perform steps necessary to verify a successful OCS installation
+    """
+    namespace = config.ENV_DATA['cluster_namespace']
+    storage_cluster_name = config.ENV_DATA['storage_cluster_name']
+
+    # Verify cluster is running
+    assert is_cluster_running(config.ENV_DATA['cluster_path'])
+
+    # Verify OCS Operator and Local Storage Operator in Succeeded phase
+    csv = ocp.OCP(kind='csv', namespace=namespace)
+    csvs = csv.get()
+    for item in csvs['items']:
+        name = item['metadata']['name']
+        log.info("Checking status of %s", name)
```

Comment:
use f-strings, same in other places

Reply:
We actually specify to let the logger do the formatting for you in our [coding guidelines](https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md)